src/features/Request/components/SelectPriceStep.tsx

```diff
@@ -0,0 +1,91 @@
+import { useMemo, useState } from 'react';
+
+const numberWithComma = (n: number) => n.toLocaleString('ko-KR');
+
+const SelectPriceStep = () => {
+  /** 입력 상태 */
+  const [count, setCount] = useState(20); // 횟수
+  const [unitPrice, setUnitPrice] = useState(20_000); // 1회 가격(원)
+
+  /** 총액 계산 */
+  const total = useMemo(() => count * unitPrice, [count, unitPrice]);
+  return (
+    <div className="relative flex w-full flex-col items-center pt-12">
+      <div className="absolute bottom-[100%] left-0 flex items-start gap-[10px]">
+        {/* 운동명, 후에 api연동 시 data넘기기 적용하고 거기서 받아오는 걸로 */}
+        <p className="font-[Pretendard Variable] text-[36px] leading-[100%] font-extrabold text-black">
+          복싱
+        </p>
+
+        {/* 주소 (예시 주소 나중에 연동해야됨) */}
+        <div className="mt-[19.5px] ml-[10px] h-[17px] w-[152px]">
+          <p className="font-[Pretendard Variable] text-[17px] leading-[100%] font-semibold text-black">
+            서울시 강서구 화곡3동
+          </p>
+        </div>
+      </div>
+
+      <h1 className="text-5xl font-extrabold">
+        희망하는 PT <span className="text-[var(--color-button)]">횟수</span> 와{' '}
+        <span className="text-[var(--color-button)]">가격</span>을 입력해주세요
+      </h1>
+
+      <section className="flex flex-col items-center gap-8 py-8">
+        {/* 횟수  */}
+        <div className="w-80 space-y-2">
+          <label className="block text-center text-sm font-semibold">횟수</label>
+          <div className="relative">
+            <input
+              type="number"
+              aria-label="PT 횟수"
+              min={1}
+              value={count}
+              onChange={(e) => setCount(Number(e.target.value))}
```

Comment:
```suggestion
              onChange={(e) => {
  const value = Number(e.target.value);
  if (!isNaN(value)) setCount(value);
}}
```

count와 unitPrice가 NaN이 되는 경우가 있을 수 있습니다.

Reply:
수정했습니다